### PR TITLE
Editing help URL and associated menu items.

### PR DESCRIPTION
**Pull request by jonnyhtw**
Hi there

This is my go at updating the ROSE help GUI URL as per issue #2410 .

All the best!

Jonny

### CODE REVIEW

**Comment by wxtim** [comment]
@jonnyhtw  - Don't worry about the test failures - these won't be related.

File: lib/python/rose/config_editor/window.py [hunk 1]
```diff
@@ -261,8 +261,7 @@ def launch_graph_dialog(self, name_section_dict):
 
     def launch_help_dialog(self, somewidget=None):
         """Launch a browser to open the help url."""
-        webbrowser.open(rose.resource.ResourceLocator.default().get_doc_url() +
-                        rose.config_editor.HELP_FILE, new=True, autoraise=True)
+        webbrowser.open('https://metomi.github.io/rose/doc/html/index.html', new=True, autoraise=True)
```
Comment (wxtim) [inline lib/python/rose/config_editor/window.py:264]:
```suggestion
        webbrowser.open(
            'https://metomi.github.io/rose/doc/html/index.html',
            new=True,
            autoraise=True
        )
```

**Review by wxtim — APPROVED** [review]
Approve subject to fixing over-long line.

**Comment by jonnyhtw** [comment]
> @jonnyhtw - Don't worry about the test failures - these won't be related.

OK thanks! I was gonna ask!... 😄

**Review by oliver-sanders — CHANGES_REQUESTED** [review]
Trying this out I'm getting traceback.

```
$ Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "~/rose/lib/python/rose/config_editor/main.py", line 2026, in <module>
    main()
  File "~/rose/lib/python/rose/config_editor/main.py", line 2022, in main
    no_warn=opts.no_warn)
  File "~/rose/lib/python/rose/config_editor/main.py", line 1882, in spawn_window
    no_warn=no_warn)
  File "~/rose/lib/python/rose/config_editor/main.py", line 285, in __init__
    self.generate_menubar()
  File "~/rose/lib/python/rose/config_editor/main.py", line 588, in generate_menubar
    widget.connect('activate', action)
AttributeError: 'NoneType' object has no attribute 'connect'
```

I think this is because the action has been renamed, but not in all places:

```console
$ git grep 'GUI Help'
demo/rose-config-edit/test/doc/test_suite:Help-> GUI Help
demo/rose-config-edit/test_suite:Help-> GUI Help
lib/python/rose/config_editor/__init__.py:TOP_MENU_HELP_GUI = "_GUI Help"
lib/python/rose/config_editor/main.py:            ('/TopMenuBar/Help/GUI Help', self.main_handle.help),
lib/python/rosie/browser/__init__.py:TOP_MENU_GUI_HELP = "_GUI Help"
lib/python/rosie/browser/main.py:                     ('/TopMenuBar/Help/GUI Help',
lib/python/rosie/browser/util.py:        <menuitem action="GUI Help"/>
lib/python/rosie/browser/util.py:                      ('GUI Help', gtk.STOCK_HELP,
```

How about renaming to "Documentation"?

**Comment by jonnyhtw** [comment]
> Trying this out I'm getting traceback.
> 
> ```
> $ Traceback (most recent call last):
>   File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
>     "__main__", fname, loader, pkg_name)
>   File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
>     exec code in run_globals
>   File "~/rose/lib/python/rose/config_editor/main.py", line 2026, in <module>
>     main()
>   File "~/rose/lib/python/rose/config_editor/main.py", line 2022, in main
>     no_warn=opts.no_warn)
>   File "~/rose/lib/python/rose/config_editor/main.py", line 1882, in spawn_window
>     no_warn=no_warn)
>   File "~/rose/lib/python/rose/config_editor/main.py", line 285, in __init__
>     self.generate_menubar()
>   File "~/rose/lib/python/rose/config_editor/main.py", line 588, in generate_menubar
>     widget.connect('activate', action)
> AttributeError: 'NoneType' object has no attribute 'connect'
> ```
> 
> I think this is because the action has been renamed, but not in all places:
> 
> ```
> $ git grep 'GUI Help'
> demo/rose-config-edit/test/doc/test_suite:Help-> GUI Help
> demo/rose-config-edit/test_suite:Help-> GUI Help
> lib/python/rose/config_editor/__init__.py:TOP_MENU_HELP_GUI = "_GUI Help"
> lib/python/rose/config_editor/main.py:            ('/TopMenuBar/Help/GUI Help', self.main_handle.help),
> lib/python/rosie/browser/__init__.py:TOP_MENU_GUI_HELP = "_GUI Help"
> lib/python/rosie/browser/main.py:                     ('/TopMenuBar/Help/GUI Help',
> lib/python/rosie/browser/util.py:        <menuitem action="GUI Help"/>
> lib/python/rosie/browser/util.py:                      ('GUI Help', gtk.STOCK_HELP,
> ```
> 
> How about renaming to "Documentation"? to avoid conflict with the existing `Help` action?

Hi @oliver-sanders, thanks a lot for that. 

I've changed all the occurrences of `GUI Help` to `Documentation` so hopefully that fixes that!?

**Comment by oliver-sanders** [comment]
Great, thanks!

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**